### ISSUE TITLE
Error when"View output files" in GPA module

### ISSUE THREAD

**Issue by aConar** [SlicerMorph/SlicerMorph#173]
Hi, I'm running the mouseskull example in GPA module (Slicer 4.11 on KDE Neon - Ubuntu - 20.04). After running GPA+PCA, the results are stored in a folder with the format DATE_TIME. But when click in "View output files" an error shows up:

![Error](https://user-images.githubusercontent.com/4622888/131505463-bf354a86-fc18-4e37-8a79-7937333a9ce7.png)

Any help will be appreciate!

Ale

**Comment by muratmaga** [comment]
I can't replicate this on windows. Can you check whether that folder is created and there are files from the terminal window. We don't have ubuntu, but will try to find to repllicate the issue.

**Comment by aConar** [comment]
Yes the folder is created and the results files are therein. Despite this, I finished the mouse example without problems. A quick note, the Slicer version bundled with Slicermorph (downloaded from seattlechildrens1.app.box.com/v/SliceMorphDownloads) don't have the "View output files" button:


![image](https://user-images.githubusercontent.com/4622888/131722105-29089105-6c6c-4739-a7c4-97c997f64e61.png)

Kind regards.

Ale

**Comment by muratmaga** [comment]
Yes, if you downloaded the packaged SlicerMorph from the seattlechildrens box address, you need to run the update mechanism of the extension manager to make sure you are using the latest version of the SlicerMorph (and all other preinstalled extensions). However, currently the extension server is down for maintenance and upgrade (You can still manually update them, but it is a bit cumbersome). If you can wait for a week or so, everything should be back to normal. You can follow it here:https://discourse.slicer.org/t/downloading-extensions-for-older-releases/19256
